model/team.py

```diff
@@ -21,11 +23,13 @@ def is_valid(team):
 
         Required fields for database to accept:
         - ``__github_team_name``
+        - ``__gtid``
```

Comment:
It _might_ be a good idea to make these part of a two-part key in DynamoDb, if both are necessary. It's not strictly necessary but it would keep everything consistent.